hvpy/core.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+def parse_response(response: requests.Response, output_parameters: OutputType):
```

Comment:
Should we consider documentation strings for these two functions in this PR?

Reply:
Yes, let's add doc strings to these.